platforms/fabric/src/main/java/com/dfsek/terra/fabric/generation/BeardGenerator.java

```diff
@@ -55,32 +55,28 @@ public BeardGenerator(StructureAccessor structureAccessor, Chunk chunk, double t
         this.pieces = new ObjectArrayList<>(10);
         int minY = chunk.getBottomY();
         int maxY = chunk.getTopY();
-        for (StructureFeature<?> structureFeature : StructureFeature.LAND_MODIFYING_STRUCTURES) {
-            for(StructureStart<?> start : structureAccessor.getStructureStarts(ChunkSectionPos.from(chunk), structureFeature)) {
-                for(StructurePiece structurePiece : start.getChildren()) {
-                    if(!structurePiece.intersectsChunk(chunkPos, 12)) continue;
-                    if(structurePiece instanceof PoolStructurePiece poolStructurePiece) {
-                        Projection projection = poolStructurePiece.getPoolElement().getProjection();
-                        if(projection == Projection.RIGID) {
-                            this.pieces.add(poolStructurePiece);
-                        }
-                        for(JigsawJunction jigsawJunction : poolStructurePiece.getJunctions()) {
-                            int k = jigsawJunction.getSourceX();
-                            int l = jigsawJunction.getSourceZ();
-                            if(k <= i - 12 || l <= j - 12 || k >= i + 15 + 12 || l >= j + 15 + 12) {
-                                continue;
-                            }
-                            maxY = Math.max(maxY, jigsawJunction.getSourceGroundY());
-                            minY = Math.min(minY, jigsawJunction.getSourceGroundY());
-                    
-                            this.junctions.add(jigsawJunction);
-                        }
-                        continue;
+        for(StructureStart start : structureAccessor.method_41035(ChunkSectionPos.from(chunk),
+                                                                  configuredStructureFeature -> configuredStructureFeature.field_37144)) {
```

Comment:
Should add comments with what those unmapped methods * fields are

Reply:
This entire class is Mojang code moved to be easier to work with. The only comment we really need is the one at the top which says where it came from.